modules/exploits/multi/misc/weblogic_deserialize_asyncresponseservice.rb

```diff
@@ -0,0 +1,167 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core/exploit/powershell'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Powershell
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name' => 'Oracle Weblogic Server Deserialization RCE - AsyncResponseService ',
+      'Description' => %q{
+        An unauthenticated attacker with network access to the Oracle Weblogic Server T3
+        interface can send a malicious SOAP request to the interface WLS AsyncResponseService
+        to execute code on the vulnerable host.
+      },
+      'Author' =>
+        [
+        'Andres Rodriguez',  # Metasploit Module - 2Secure (@acamro, acamro[at]gmail.com)
+        ],
+      'License' => MSF_LICENSE,
+      'References' =>
+        [
+          ['CNVD-C', '2019-48814'],
```

Comment:
@wvu-r7 @acamro can you toss in a couple more references here please?

Oracle's advisory: https://www.oracle.com/technetwork/security-advisory/alert-cve-2019-2725-5466295.html 
CVE: CVE-2019-2625

Thanks!

Reply:
Just a note @todb-r7, CVE is actually CVE-2019-2725.

Reply:
Yep, this is the final CVE, thanks.